### PR TITLE
Add Harmony backend

### PR DESCRIPTION
**Pull request by pontaoski**

### CODE REVIEW

**Comment by 42wim** [comment]
Can you give some more information? What is Harmony?

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "bridge/harmony/protocol"]
+	path = bridge/harmony/protocol
+	url = https://github.com/harmony-development/protocol.git
```
Comment (qaisjp) [inline .gitmodules:3]: Go tooling doesn't have great support for submodules - will this make it harder to build matterbridge? At the moment you can just `go get` it and it'll build and work.

Comment (pontaoski) [reply .gitmodules]: got rid of the submodule

**Comment by pontaoski** [comment]
> Can you give some more information? What is Harmony?

Harmony is a new-ish chat protocol being developed at https://github.com/harmony-development

**Comment by pontaoski** [comment]
opening a new one

**Pull request closed without merging**